spec/mailboxes/ticket_mailbox_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe TicketMailbox, type: :mailbox do
+  let(:body) { Faker::Lorem.paragraphs(number: 2).join(',') }
+  let(:subject) { Faker::Internet.name }
+
+  context 'with valid params' do
+    let(:user) { create(:user) }
+
+    it 'marks email as delivered when the from receipt is monterrey' do
+      mail = Mail.new(from: user.email,
```

Comment:
I guess that to avoid some dup. the mail creation could be extracted to a `let(:mail)` and then define from, to, subject, body params, too.